examples/datadog-sink/main.tf

```diff
@@ -62,6 +62,7 @@ module "log_export" {
   log_sink_name          = "test-datadog-sink"
   parent_resource_id     = var.parent_resource_id
   parent_resource_type   = "project"
+  filter                 = var.filter
```

Comment:
Thanks for the PR @taman9333 
We generally use examples as ways to showcase different usecases with a module, but our examples are not meant to be used directly as a module. In this case, as this is an example we prefer to keep it simpler without introducing new variable. For that reason I am closing this PR.

Reply:
Got it, I thought `log-export_example_datadog-sink` module could be used directly as a module 👍